tests/contrib/django/test_connection.py

```diff
@@ -30,7 +26,7 @@ def test_connection(self):
 
         span = spans[0]
         eq_(span.name, 'sqlite.query')
-        eq_(span.service, 'defaultdb')
+        eq_(span.service, 'django-defaultdb')
```

Comment:
Thinking in general, this will probably break current applications since developers may look at the previous `defaultdb` service instead of the new one. So the idea is not to introduce a breaking change here, but to provide a way to configure that.

What if we keep the service as is (so `defaultdb`) but we expose a `settings.DEFAULT_DB_SERVICE` that can be used to override that value?